csm_web/csm_web/settings.py

```diff
@@ -152,6 +152,16 @@
 
 USE_TZ = True
 
+# AWS environment variables for S3 resource storage
+AWS_ACCESS_KEY_ID = os.environ.get("AWS_ACCESS_KEY_ID")
```

Comment:
make sure these are added to the heroku configs. try the was stuff out on staging first. run django shell on staging to make migrations if you haven't already